tox.ini

```diff
@@ -16,3 +15,12 @@ deps =
     django-19: Django>=1.9,<1.10
     django-110: Django>=1.10,<1.11
```

Comment:
Maybe that would be a good idea to also test the last version of django in order to detect incompatibilities early.